scripts/plugins/storage/postgres/upgrade/75.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE fledge.streams ADD COLUMN last_objects jsonb  NOT NULL DEFAULT '{"Readings":0,"Stats":0,"Audit":0}'::jsonb;
```

Comment:
The is not preserving the current point reached in the readings being sent north. This will be an issue for upgrade of a running system.